chaoslib/litmus/platform/gke/node_cpu_consumption.yml

```diff
@@ -25,39 +25,39 @@
         message: "Injecting {{ c_experiment }} chaos on {{ node_name }} node"
       when: "c_engine is defined and c_engine != ''"
       
-    ## CREATING  A DAEMONSET  FOR CPU-HOG
-    - name: Creating a DaemonSet for cpu-hog
+    ## CREATING  A Job  FOR CPU-HOG
+    - name: Creating a Job for cpu-hog
       k8s:
         state: present
         namespace: "{{ a_ns }}"
-        src: /chaoslib/litmus/platform/gke/node-cpu-hog-daemonset.yml
-      register: cpu_hog_ds_result
+        src: /chaoslib/litmus/platform/gke/node-cpu-hog-job.yml
+      register: cpu_hog_job_result
 
-    ## CHECKING THE STATUS OF DAEMONSET
-    - name: Checking the status of DaemonSet
+    ## CHECKING THE STATUS OF JOB Pod
+    - name: Checking the status of Job Pod
       k8s_facts:
         kind: Pod
         namespace: "{{ a_ns }}"
         label_selectors:
           - app=cpu-hog
-      register: pod_ds
-      until: "{{pod_ds | json_query('resources[*].status.phase') | unique | length==1}} and {{pod_ds | json_query('resources[0].status.phase') == 'Running'}}"
+      register: pod_job
+      until: "{{pod_job | json_query('resources[*].status.phase') | unique | length==1}} and {{pod_job | json_query('resources[0].status.phase') == 'Running'}}"
       delay: 2
       retries: 90
 
-    ## GETTING THE POD NAME OF DS FROM APPLICATION NODE
-    - name: Getting the pod name of DS from application node
+    ## GETTING THE POD NAME OF JOB FROM APPLICATION NODE
+    - name: Getting the pod name of Job from application node
```

Comment:
In this task you are getting name of the job pod but in the previous task you are checking the status of job-pod? Can you please reverse the order and use this extracted name for checking the status.